pkgs/development/haskell-modules/configuration-ghc-9.12.x.nix

```diff
@@ -57,6 +57,25 @@ self: super: {
   unix = null;
   xhtml = null;
 
+  doctest = lib.pipe super.doctest [
+    (overrideSrc {
+      # Not yet on hackage: https://github.com/sol/doctest/pull/462
+      src = pkgs.fetchFromGitHub {
+        owner = "sol";
+        repo = "doctest";
+        rev = "77373c5d84cd5e59ea86ec30b9ada874f50fad9e";
+        hash = "sha256-nPfVDnHlKL2DYo9+IDKxA2AvRnjLQ8kr8YGHHD9APvM=";
+      };
+    })
```

Comment:
This is likely to grow stale. I’d like to see either a patch or an assert.

Reply:
I had originally attempted a patch of the relevant commit on the latest release but even that was too old and got conflicts. 
Added a warning